__tests__/app.test.js

```diff
@@ -431,6 +431,12 @@ describe("GET /api/articles?sortby&&order&&topic", () => {
 });
 describe("DELETE /api/comments/:comment_id", () => {
   test("status:204, responds with an empty response body", () => {
-    return request(app).delete("/api/comments/1").expect(204);
+    return request(app).delete("/api/comments/5").expect(204);
+  });
+  test("status:400, responds with bad request when invalid input", () => {
+    return request(app).delete("/api/comments/notnumber").expect(400);
+  });
+  test("status:404, responds with not found when comment id not found", () => {
+    return request(app).delete("/api/comments/999").expect(400);
```

Comment:
should be expecting a 404 here for comment id not found (still down as 400 in your expect() on line 440)